blog/content/post/2021-06-14-fwupd-freebsd-status.md

```diff
@@ -0,0 +1,1499 @@
+---
+title: fwupd on FreeBSD - Status Update
+abstract: 'An update on the current status of the BSD port of fwupd.'
```

Comment:
@mkopec @Asiderr it would be more appealing to have abstract with at least 2-3 sentences